htdocs/admin/invoice.php

```diff
@@ -758,6 +759,17 @@
 print ajax_combobox("chq", array(), 0, 0, 'resolve', '-2');
 
 print "</td></tr>";
+
+// Structured communication
+// France / Germany / Netherlands do not use this structured system of invoice's payment
+if ($mysoc->country_code != 'FR' && $mysoc->country_code != 'DE' && $mysoc->country_code != 'NL') {
```

Comment:
Hi @aspangaro could you change this line to 

`if ($mysoc->country_code = 'BE') {`

as it only applies to Belgium for the moment

Reply:
I've read that this doesn't apply in France, Germany and the Netherlands (cf link to ISO 20022 standard from the forum), so the activation option will be available for Belgium and other countries, but not for the three mentioned above.

Reply:
The use of  ISO 20022 standard will be mandatory for all SEPA countries (including France, Germany and the Netherlands)  by November 2025 and changes will have to be made to Dolibarr as well, but it's not the issue here.

https://www.securities-services.societegenerale.com/en/insights/views/news/adoption-of-the-swift-iso-20022-standard-for-corporate-actions/

https://www.ubaf.fr/en/news/iso-20022-adoption-and-migration-cross-border-payments-and-reporting-plus

As stated above the structured communication payment reference only applies to Belgium for the moment and is sort of the equivalent to Finland's RF payment reference which is already implemented

https://github.com/Dolibarr/dolibarr/blob/26bb977006ce803363dcbc8c529bce61461f49f9/htdocs/core/lib/functions_fi.lib.php#L37

Reply:
Thanks for all the information @sonikf I move the function in specific file for Belgium.

Also, in admin, the option is only seen by our Belgian friends